libgadget/slotsmanager.h

```diff
@@ -41,24 +41,18 @@ struct bh_particle_data {
 
     MyFloat Mass;
     MyFloat Mdot;
-    MyFloat FeedbackWeightSum;
     MyFloat Density;
-    MyFloat Entropy;
-    MyFloat Pressure;
-    MyFloat SurroundingGasVel[3];
-    MyFloat FormationTime;		/*!< formation time of black hole. */
-
-    MyFloat accreted_Mass;
-    MyFloat accreted_BHMass;
-    MyFloat accreted_momentum[3];
+    MyFloat FormationTime;  /*!< formation time of black hole. */
 
     int JumpToMinPot;
```

Comment:
Is this still used?

Reply:
It is never actually used, but it is written to disc - could be useful for some post-processing analyses.